index.html

```diff
@@ -6,18 +6,30 @@
   <script src="https://use.fontawesome.com/2506d2b99f.js"></script>
   <link type="text/css" rel="stylesheet" href="style/reset.css" />
   <link type="text/css" rel="stylesheet" href="style/style.css" />
+  <link type="text/css" rel="stylesheet" href="style/layout.css" />
+  <link type="text/css" rel="stylesheet" href="style/module.css" />
   <title>Life of Brian (WebDev)</title>
+  <script id="project-template" type="text/x-handlebars-template">
+    <section id='tab-content'>
+      <article class="template">
+        <h1>{{projectTitle}}</h1>
+        <p>{{projectlastUpdated}}</p>
+        <p>{{gitHubURL}}</p>
+        <p>{{projectDescription}}</p>
+      </article>
+    </section>
+  </script>
 </head>
 <body>
   <header class="clearfix">
      <i class="fa fa-bars" class="hamberger" aria-hidden="true"></i>
      <nav class="full-nav">
       <div class="icons">
         <ul>
-          <li><i class="fa fa-github-alt" aria-hidden="true"></i></li>
-          <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
-          <li><i class="fa fa-user-circle-o" aria-hidden="true"></i></li>
-          <li><i class="fa fa-home" aria-hidden="true"></i></li>
+          <a href="#" onClick="window.open('https://github.com/Xanture', '_blank')"><li><i class="fa fa-github-alt" aria-hidden="true"></i></li></a>
```

Comment:
You should never use `onclick` anymore. It is a leftover artifact of bygone days. If you want the links to open in a new tab, you can use the [`target`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) attribute. If you want some JS to run you can bind an event to it with a selector.